### PR TITLE
Adding my part of the story "blackout"

### PR DESCRIPTION
**Pull request by wongkmatthew**

### CODE REVIEW

File: english/blackout/blackout.md [hunk 1]
```diff
@@ -0,0 +1 @@
+You walk down the tunnel and feel faint. You blackout and right before you hit the floor someone catches you.
```
Comment (casey-collab) [inline english/blackout/blackout.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by wongkmatthew** [comment]
Please review this, @casey-collab

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by wongkmatthew** [comment]
Please review this, @casey-collab

**Comment by casey-collab** [comment]
Looks great! :lollipop:

**Pull request merged**